src/main/java/io/appform/dropwizard/actors/actor/MessageHandlingFunction.java

```diff
@@ -20,6 +20,6 @@
  *
  */
 @FunctionalInterface
-public interface MessageHandlingFunction<T,R> {
-    R apply(T param) throws Exception;
+public interface MessageHandlingFunction<T, X, R> {
```

Comment:
MessageMetadata type is fixed. There is no point in adding it as a parameter.

Reply:
removed.